app/components/chat_message/chat_message.html.erb

```diff
@@ -120,7 +83,13 @@
             <%= I18n.t('components.chat_message.move') %>
           <% end %>
         <% end %>
-        
+
+        <% if message.sent_from_contributor? %>
+          <%= c 'button', link: conversations_contributor_path(id: message.contributor, reply_to: message.id, anchor: "chat-form" ), style: :inline do %>
+            <%= c 'icon', icon: 'reply-arrow', styles: [:inline] %>
+            <%= I18n.t('components.chat_messages_group.reply') %>
+          <% end %>
+        <% end %>
```

Comment:
This is not really greast UX, in my opinion. This lead to an entire reload of the page and when there are a lot of messages that's slow. It's not a trivial fix though.

If I understand correctly, the reason that you don't use JS to simply scroll to the chat form is that you set the `reply_to` in the url, which sets the `@reply_to` instance variable, which is passed into the chat form component and instructs the component to show the heading and to add the id to the hidden tag so that the message can be attributed to the correct request.

I can imagine fixing this by having an endpoint that renders the chat form component, which could be used to lazy load the chat form when the url is hit, then sending a request to that endpoint when the click nachfragen button and replace the component. I think this would be a much better user experience; however, we could discuss and can certainly be implemented in a next step.

Reply:
Of course there is always room for improvement. I tend to hold the things simple stupid as long as there is a real need to do otherwise..

Reply:
> Of course there is always room for improvement. I tend to hold the things simple stupid as long as there is a real need to do otherwise..

I assume this is in regards to the UX of reloading the whole page? For me this makes using the feature with production-style data very difficult, but maybe others disagree and we could get it out to the clients to see how they feel. Maybe my solution isn't the best either. I could imagine the link leading to the exisiting Chat component, which I believe would display simple the one message the client wants to reply to along with the chat form. I'm sure there are many other options.